frontend/src/styles/global.css

```
@@ -1,7 +1,41 @@
+  --color-text-muted: #526272;
+  --color-border: #d6e0e8;
+  --color-border-strong: #8c9baa;
+  --color-focus: rgb(33 150 243 / 24%);
+  --color-success: #18794e;
+  --color-danger: #c62828;
+  --color-danger-border: #ef9a9a;
+  --color-danger-text: #9f1d1d;
+  --radius-control: 9px;
+  --radius-card: 14px;
+  --shadow-card: 0 10px 28px rgb(13 71 161 / 8%);
+  color: var(--color-text);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)